src/main/resources/static/scripts/route.js

```diff
@@ -70,12 +44,36 @@ Application.run(function ($http, $q) {
             templateUrl: "views/planByBuyer.html",
             controller: "planByBuyerController"
         })
-        .when('/', {
+        .when('/buyer/dashboard', {
             templateUrl: "views/dashboard.html",
             controller: "dashboardController"
         })
         .when('/admin/dashboard', {
             templateUrl: "views/adminDashboard.html",
-            controller: "adminDashboardController"
+            controller: "adminDashboardController",
+            resolve: {
+                factory: checkRouting
```

Comment:
Do we need this factory only for `admin/dashboard` path?

Reply:
for each route that would have 2 or more usages
i think it would be only for dashboard